client/src/Hooks/useUrlParams/useUrlParams.tsx

```
@@ -0,0 +1,51 @@
+import { useSearchParams } from 'react-router-dom';
+import { useCallback } from 'react';
+import { SearchParams } from '../../interfaces/Search.interface';
+
+export function useUrlParams() {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  // Get initial values from URL
+  const initialPage = parseInt(searchParams.get('page') || '1');
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)